corehq/apps/es/tests/test_client.py

```diff
@@ -1795,6 +1820,354 @@ def test_create_document(self):
         )
 
 
+@es_test(requires=[user_adapter], setup_class=True)
+class TestFromMultiInElasticUser(TestCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.domain = 'from-python-tests'
+        cls.domain_obj = create_domain(cls.domain)
+        cls.addClassCleanup(cls.domain_obj.delete)
+        cls.user = CommCareUser.create(
+            username="rock", domain=cls.domain, password="***********",
+            created_by=None, created_via=None
+        )
+        cls.web_user = WebUser.create(
+            username='webman', domain=cls.domain, password="***********",
+            created_by=None, created_via=None
+        )
+
+    @classmethod
+    def tearDownClass(cls):
+        delete_all_users()
+        super().tearDownClass()
```

Comment:
nit (I'm not really sure if this gains anything functionally, but it's a little cleaner)

```suggestion
        cls.addClassCleanup(delete_all_users)
```

Reply:
Yep. Thanks for the suggestion.